### DISCUSSION TITLE
Magic search in shared albums

### DISCUSSION THREAD

**Discussion by xxblx** [ente-io/ente#2557] [Enhancements]
Hi there, 

Would it be possible to add indexing of content in shared albums that was uploaded by someone else and a support for magic search in shared albums? 

Use case. My wife and I use Ente Photos. Because I did an initial upload of our existing photo library, tens of thousands of photos were indexed on my devices and magic search works great for me, but even though I shared all albums with my wife, magic search doesn't work for her on content that I uploaded. In other words, magic search only works for her on content she uploaded, and magic search only works for me on content I uploaded. And because I did the initial upload of photos to Ente, most of our photo library is not magic-searchable for her. We both use iPhones, I don't know if it would be the same behavior on Android or not. 

Therefore, it would be amazing if Ente could index photos in a shared album uploaded by other users, if you have access to the album, so that the content of the album would be searchable (magic search). For now, all the smart search features of Ente seem to work only for the uploader of the photos, while other users can only search shared albums by album name, location and browse  shared albums manually. 

Thanks,
Oleg

p.s. Thank you for developing this amazing app! It's been 4 months since I subscribed to a paid plan. Other than the topic of discussion here and #787, Ente does everything I need and works perfectly well.

**Comment by ua741** [comment]
Our team discussed this just last week :). 
Once Faces Beta is out, we will start working on sharing the indexes for shared files.

**Reply by xxblx** [reply]
Awesome. Thank you!
